acct-user/asterisk/asterisk-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for asterisk"
+ACCT_USER_ID=242
+ACCT_USER_HOME=/var/lib/asterisk
```

Comment:
@mjo I need your guidance with these.

Reply:
Wrong person? :)

Reply:
https://archives.gentoo.org/gentoo-dev/message/ce73d8d5170b1f6ec04e19ea3bdae267

Got some objections, but this one received no objections:

https://archives.gentoo.org/gentoo-dev/message/8b139b0c345da847f1f0bd95abb850fb

Reply:
> Wrong person? :)

@mjo umm... seems so, sorry about the noise :)

Let's try this again: @orlitzky can we have your word on **defining these variables**, 
@jkroonza I don't doubt the GID+UID numbers are good, but I've seen a lot of ... preaching about defining ACCT_USER_HOME, however I haven't had time to study it closer by myself. So I'm consulting @orlitzky to explain it for both of us here :P

Reply:
https://www.gentoo.org/glep/glep-0081.html#home-directory-ownership

So from that I understand:  Do not set ACCT_USER_HOME unless specifically required.

So I updated my test system to home folder /dev/null, and stopped and started asterisk with no problems.  So dropped from update.

Reply:
Yep that's right. I've given up on trying to change the devmanual, so in the meantime I've [posted the guidelines on my website](http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml) for easy copy/paste:

tl;dr use the defaults if possible